include/ddc/kernels/splines/spline_builder.hpp

```diff
@@ -142,6 +183,22 @@ class SplineBuilder
     int compute_offset(interpolation_domain_type const& interpolation_domain);
 
 public:
+    /**
+     * @brief Build a SplineBuilder acting on batched_interpolation_domain.
+     * 
+     * @param batched_interpolation_domain The domain on which the interpolation points are defined.
+     * @param cols_per_chunk A hyperparameter used by the slicer (internal to the solver) to define the size
+     * of a chunk of right-hand-sides of the linear problem to be computed in parallel (chunks are treated
+     * by the linear solver one-after-the-other).
+     *
+     * This value is optional. If no value is provided then the default value is chosen by the requested solver.
+     * @param preconditionner_max_block_size A hyperparameter used by the slicer (internal to the solver) to
+     * define the size of a block used by the Block-Jacobi preconditioner.
+     *
+     * This value is optional. If no value is provided then the default value is chosen by the requested solver.
+     *
```

Comment:
The spacing here seems odd and makes it hard to read. I would recommend:
```suggestion
     * @param batched_interpolation_domain The domain on which the interpolation points are defined.
     *
     * @param cols_per_chunk A hyperparameter used by the slicer (internal to the solver) to define the size
     * of a chunk of right-hand-sides of the linear problem to be computed in parallel (chunks are treated
     * by the linear solver one-after-the-other).
     * This value is optional. If no value is provided then the default value is chosen by the requested solver.
     *
     * @param preconditionner_max_block_size A hyperparameter used by the slicer (internal to the solver) to
     * define the size of a block used by the Block-Jacobi preconditioner.
     * This value is optional. If no value is provided then the default value is chosen by the requested solver.
     *
```